apps/web/app/_lib/donera.ts

```diff
@@ -58,5 +58,7 @@ export const getDoneraDapp = (): DoneraDapp => {
   return donera;
 };
 
-export const cidToUrl = (cid?: string | null) =>
-  cid ? `${process.env.NEXT_PUBLIC_IPFS_GATEWAY_URL}/${cid}` : "";
+export const ipfsUrlToGateway = (ipfsUrl: string) => {
+  const cid = ipfsUrl.split("//")[1];
```

Comment:
verify cid is valid, this could be set to an arbitrary value and we would render the image

Reply:
No we wouldnt, we always assume this is a url path that is appended to the gateway, should be safe